pom.xml

```diff
@@ -13,6 +13,11 @@
   <name>Jenkins jQuery plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.625</jenkins.version>
+    <java.level>6</java.level>
```

Comment:
Why is better 1.625.3 than 1.625? Does that not result in the possibility of failure in cores like 1.625, 1.626 and so on? I mean, an LTS can contain changes not included on weekly versions, resulting in an effective bump to the weekly version which has the latest fix added to 1.625.3 LTS